src/main/java/core/basesyntax/strategy/FruitStrategyImpl.java

```diff
@@ -0,0 +1,33 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.model.Fruit;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.function.BiFunction;
+
+public class FruitStrategyImpl implements FruitStrategy {

```

Comment:
It is an incorrect strategy implementation. Let’s watch the lesson again and implement it by example. If you have any difficulties let’s discuss it in the chat ;)